textworld/render/render.py

```diff
@@ -320,7 +317,12 @@ def take_screenshot(url: str, id: str='graph2'):
     :param id: ID of DOM element.
     :return: Image object.
     """
-    from PIL import Image
+    try:
+        from PIL import Image
+        from selenium import webdriver
+        from selenium.webdriver.chrome.options import Options
+    except ImportError:
+        raise ImportError('Visualization dependencies not installed. Try running `pip install textworld[vis]`')
```

Comment:
I'm not convinced this is better than just letting the original `ImportError` bubble up